cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgstautau.json

```diff
@@ -0,0 +1,107 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
```

Comment:
"CMS-Derived-Datasets" is the value to use for datasets... I can change during merge.